src/common/errors/errors.ts

```diff
@@ -1,17 +1,23 @@
-export const ERROR_MESSAGES = {
+import {ErrorCode} from "./enum/error-code";
+
+export const ERRORS = {
   auth: {
     notAuthorized: {
+      code: ErrorCode.NOT_AUTHORIZED,
       message: 'not authorized',
     },
     invalidMessage: {
+      code: ErrorCode.INVALID_MESSAGE,
       message: 'invalid message',
     },
     forbidden: {
+      code: ErrorCode.FORBIDDEN,
```

Comment:
Do we need a description for default server errors? look like not, We need to follow one approach. Let's discuss